Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apt-get update -y && \
 # renovate: datasource=npm depName=renovate
 ENV RENOVATE_VERSION=35.31.0
 
-RUN npm install -g renovate@${RENOVATE_VERSION} --ignore-scripts && \
+RUN npm install -g renovate@${RENOVATE_VERSION} && \
```

Comment:
Is there no other way around that?
I liked the security aspects of `--ignore-scripts`

Reply:
An option could be to keep `--ignore-scripts` and explicitly call `prepare` script

Reply:
I am happy with both ways.

In either case we must rely on that the `prepare` script does not contain malicious code, where as the explicit call of the `prepare` script is less of a risk than running all available life cycle scripts.

Reply:
I couldn't find a clean way to install RE2, with Renovate being installed globally. Therefore I would suggest to revert back to run scripts during installation.